common/src/main/java/com/microsoft/identity/common/internal/providers/azureactivedirectory/AzureActiveDirectoryAccount.java

```diff
@@ -110,6 +110,9 @@ private String getDisplayableId(Map<String, String> claims) {
             return claims.get(AzureActiveDirectoryIdTokenClaims.UPN);
         } else if (!StringExtensions.isNullOrBlank(claims.get(StandardIdTokenClaims.EMAIL))) {
             return claims.get(StandardIdTokenClaims.EMAIL);
+        } else if (!StringExtensions.isNullOrBlank(claims.get(AzureActiveDirectoryIdTokenClaims.PREFERRED_USERNAME))) {
+            // TODO this is for the v2 app model -- does this need to go somewhere separate?
```

Comment:
@shoatman I've added `PREFERRED_USERNAME` -- this is a v2 claim. Currently I've stashed it inside of `AzureActiveDirectoryIdTokenClaims` but that's probably not the best place for it, since that's v1

Reply:
Resolved